bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/aQGC_WZA_LO/WZA_WToLNu_LOaQGC_reweight_card.dat

```diff
@@ -0,0 +1,4636 @@
+change rwgt_dir rwgt
+
+
+launch
+    set FS0 0
+    set FS1 0
+    set FS2 0
+    set FM0 0
+    set FM1 0
+    set FM2 0
+    set FM3 0
+    set FM4 0
+    set FM5 0
+    set FM6 0
+    set FM7 0
+    set FT0 0
+    set FT1 0
+    set FT2 0
+    set FT3 0
+    set FT4 0
+    set FT5 0
+    set FT6 0
+    set FT7 0
+    set FT8 0
+    set FT9 0
+
+
+
+
+launch
+	set FS0 0
+	set FS1 0
+	set FS2 0
+	set FM0 -4e-10
+	set FM1 0
+	set FM2 0
+	set FM3 0
+	set FM4 0
+	set FM5 0
+	set FM6 0
+	set FM7 0
+	set FT0 0
+	set FT1 0
+	set FT2 0
+	set FT3	0
+	set FT4	0
+	set FT5 0
+	set FT6 0
+	set FT7 0
+	set FT8 0
+	set FT9 0
+
+
+launch
+	set FS0 0
+	set FS1 0
+	set FS2 0
+	set FM0 -3e-10
+	set FM1 0
+	set FM2 0
+	set FM3 0
+	set FM4 0
+	set FM5 0
+	set FM6 0
+	set FM7 0
+	set FT0 0
+	set FT1 0
+	set FT2 0
+	set FT3	0
+	set FT4	0
+	set FT5 0
+	set FT6 0
+	set FT7 0
+	set FT8 0
+	set FT9 0
+
+
+launch
+	set FS0 0
+	set FS1 0
+	set FS2 0
+	set FM0 -2e-10
+	set FM1 0
+	set FM2 0
+	set FM3 0
+	set FM4 0
+	set FM5 0
+	set FM6 0
+	set FM7 0
+	set FT0 0
+	set FT1 0
+	set FT2 0
+	set FT3	0
+	set FT4	0
+	set FT5 0
+	set FT6 0
+	set FT7 0
+	set FT8 0
+	set FT9 0
+
+
+launch
+	set FS0 0
+	set FS1 0
+	set FS2 0
+	set FM0 -1e-10
+	set FM1 0
+	set FM2 0
+	set FM3 0
+	set FM4 0
+	set FM5 0
+	set FM6 0
+	set FM7 0
+	set FT0 0
+	set FT1 0
+	set FT2 0
+	set FT3	0
+	set FT4	0
+	set FT5 0
+	set FT6 0
+	set FT7 0
+	set FT8 0
+	set FT9 0
+
+
+launch
+	set FS0 0
+	set FS1 0
+	set FS2 0
+	set FM0 -5e-11
+	set FM1 0
+	set FM2 0
+	set FM3 0
+	set FM4 0
+	set FM5 0
+	set FM6 0
+	set FM7 0
+	set FT0 0
+	set FT1 0
+	set FT2 0
+	set FT3	0
+	set FT4	0
+	set FT5 0
+	set FT6 0
+	set FT7 0
+	set FT8 0
+	set FT9 0
+
+
+launch
+	set FS0 0
+	set FS1 0
+	set FS2 0
+	set FM0 2e-11
```

Comment:
I think this weight is supposed to -2e-11, as 2e-11 appears in the next launch command. The same thing happens for the next few EFT operators as well, and I'm worried there could be similar errors that we won't catch because it's quite hard to review this. Did you generate this file with a script? Maybe you could also commit that so we can review it and it can be used by other people doing similar requests in the future

Reply:
Sorry, I just saw you linked the cards used to generate this- it looks like this is an issue for FM0-3, but the others are fine. Personally I'd say it's worth commiting that too, so it can be reused by future requests for the same process, but I don't know what the policy on this is

Reply:
> Sorry, I just saw you linked the cards used to generate this- it looks like this is an issue for FM0-3, but the others are fine. Personally I'd say it's worth commiting that too, so it can be reused by future requests for the same process, but I don't know what the policy on this is

Thank you very much. I added script to make reweight card. Also modified FM0-3 error